bin/ruby-build

```diff
@@ -367,6 +367,8 @@ fetch_tarball() {
       fi
     elif [ -n "$RUBY_BUILD_MIRROR_PACKAGE_URL" ]; then
       mirror_url="$RUBY_BUILD_MIRROR_PACKAGE_URL"
+    elif [ -n "$RUBY_BUILD_MIRROR_CMD" ]; then
+      mirror_url="$("$RUBY_BUILD_MIRROR_CMD" "$package_url" "$checksum")"
```

Comment:
This should use `command` to ensure that the name executable is called even if it coincides with a name of a bash builtin or a shell function defined in this file
```suggestion
      mirror_url="$(command "$RUBY_BUILD_MIRROR_CMD" "$package_url" "$checksum")"
```